### DISCUSSION TITLE
Call to undefined method Saloon\PaginationPlugin\OffsetPaginator@anonymous::setStartPage()

### DISCUSSION THREAD

**Discussion by RhysLees** [saloonphp/saloon#351] [Q&A] [answered]
I have a paginated request that i wish to get page 2 of. i figured that i would need to set the start page and max pages and it works fine locally, but it seems to fail in CI saying that setStartPage is undefined.

Call to undefined method Saloon\PaginationPlugin\OffsetPaginator@anonymous::setStartPage()

at tests/Feature/Requests/Search/GetSearchRequestTest.php:

```php
$paginator = $this->connector->paginate($paginatorRequest);

$paginator->setPerPageLimit(5)
  ->setStartPage(2)
  ->setMaxPages(2);
```

Am i doing something wrong here or is this a bug?
Not sure as to why its working locally but not in ci.

**Comment by Sammyjo20** [comment, marked as answer]
Hey @RhysLees!

That method definitely exists in the pagination plugin! Does your CI use `composer install --prefer-lowest` or anything like that? If your CI does, then just change the pagination plugin to the latest minor version. For example, the `setStartPage` was only added in Saloon `^2.1` so you need to set that to be your minimum.

Let me know if that doesn't work.

**Reply by RhysLees** [reply]
Ah man, not sure how i missed that, thankyou!

**Reply by Sammyjo20** [reply]
Wooo, so glad that fixed it!
